app.js

```diff
@@ -0,0 +1,47 @@
+import express from 'express';
+import bodyParser from 'body-parser';
```

Comment:
Uninstall the `bodyParser` module.

Express now incorporates the `bodyParser` functionality itself. As such, the following will become:

`app.use(express.urlencoded({ extended: false }));`
`app.use(express.json());`

Reply:
Okay. will do that now

Reply:
I have uninstall the body-parser